src/main/java/com/example/barbertime/Entities/User.java

```diff
@@ -0,0 +1,117 @@
+package com.example.barbertime.Entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.persistence.*;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+import java.util.List;
+
+@Entity
+@Table(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "name")
+    @NotBlank(message = "name is required")
+    @Size(min = 2, message = "name must be more than 2 characters!")
+    private String name;
+
+    @Column(name = "email")
+    @NotBlank(message = "email is required")
+    @Email(message = "Must add @ sign")
+    private String email;
+
+    @Column(name = "password")
```

Comment:
Make sure you are using the validation correctly, having a password made up from 3 characters is not safe